src/tests/state/approvalRequests/reducers.test.js

```diff
@@ -55,20 +56,23 @@ describe('Approval requests reducer', () => {
   });
 
   it('should handle RECIPE_HISTORY_RECEIVE', () => {
-    const appRequest = {
-      id: 'test',
-      arbitrary_data: 123,
+    const reducedApprovalRequest = {
+      ...approvalRequest,
+      approver_id: approvalRequest.approver ? approvalRequest.approver.id : null,
```

Comment:
Why this test? Don't we know at this point what the test data is?

Reply:
I'm not sure I copied it over from above, but I'll do some digging and remove if unnecessary.